apps/web/styles/globals.css

```diff
@@ -64,6 +64,7 @@
   --cal-border-emphasis: #575757;
   --cal-border: #444444;
   --cal-border-subtle: #2b2b2b;
+  --cal-border-booker: #2b2b2b;
```

Comment:
@sean-brydon I added this only for dark mode. I believe light mode variables are automatically generated by tailwind and they aren't required?

Also, I found a similar file colors.css which isn't being used probably and has the same variables defined there. I didn't make a change intentionally there.

Reply:
Youre right - however if possible please define them in globals.css too  - Makes them easier to manage i believe.